pkg/controllers/gracefuleviction/evictiontask.go

```diff
@@ -108,6 +108,11 @@ func nextRetry(tasks []workv1alpha2.GracefulEvictionTask, timeout time.Duration,
 	retryInterval := timeout / 10
 
 	for i := range tasks {
+		if tasks[i].SuppressDeletion != nil {
+			if *tasks[i].SuppressDeletion {
```

Comment:
Do we need the inner if-judge? I feel like it's redundant.

Reply:
I think we do not need to judge *task[i].SuppressDeletion = true or false. I think It should not be false and we can reduce computational complexity.

Reply:
https://github.com/karmada-io/karmada/blob/master/pkg/controllers/gracefuleviction/evictiontask.go#L52-L54

 I think it would be easier to understand if we keep it consistent with what's here.

Reply:
It differs in two places. L52-L54 needs to judge *task[i].SuppressDeletion to determine whether the task need to kept. When calculating the duration, we just need to know whether it's a suppression or gracious task.

Reply:
Updated.